app/controllers/api/v2/permissions_controller.rb

```diff
@@ -34,6 +35,14 @@ def resource_types
         @total = @resource_types.size
         render :resource_types, :layout => 'api/v2/layouts/index_layout'
       end
+
+      private
+
+      def parameter_deprecation
```

Comment:
My bad, forgot to include the before_filter to call this method which should be present now.